### PR TITLE
Update java-app-best-practices.md

### PR DESCRIPTION
**Pull request by sev7ndayyoo**
Only configuring the url of jdbc is invalid for streaming queries under the mybatis framework

### What is changed, added or deleted? (Required)

Only configuring the url of jdbc is invalid for streaming queries under the mybatis framework

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: best-practices/java-app-best-practices.md [hunk 1]
```diff
@@ -255,8 +255,9 @@ MyBatis 的 Mapper 中支持两种参数：
 
 #### Streaming 结果
 
-前面介绍了在 JDBC 中如何使用流式读取结果，除了 JDBC 相应的配置外，在 MyBatis 中如果希望读取超大结果集合也需要注意：
+前面介绍了在 JDBC 中如何使用流式读取结果，在 MyBatis 中如果希望读取超大结果集合也需要注意：
 
+- 只在 JDBC 的 url 配置 defualtFetchSize ，对于 MyBatis 框架下使用的流式查询是无效的
```
Comment (djshow832) [inline best-practices/java-app-best-practices.md:260]:
```suggestion
- 只在 JDBC 的 url 配置 `defaultFetchSize`，对于 MyBatis 框架下使用的流式查询是无效的
```

Comment (djshow832) [reply best-practices/java-app-best-practices.md:260]: Is there any document on this conclusion?

**Comment by TomShawn** [comment]
@sev7ndayyoo Could you please provide the source on which your change is based? Thanks!

**Comment by TomShawn** [comment]
@sev7ndayyoo To better manage this repo, I'll close this PR for now. If you have any source to provide or further change to make, you are welcome to reopen this PR, thanks!

**Pull request closed without merging**